### DISCUSSION TITLE
searching for a list of entries in a specific directory

### DISCUSSION THREAD

**Discussion by DanielsBerlin** [BurntSushi/ripgrep#2383] [General] [answered]
hi,  and thanks for the program.
suppose that i have a list which contains 100 entries, and i have a directory that contains some txt files and some zip files. so i want to search the list inside that directory in a way that when any of the entries are hit in that directory , the program needs to list those matching entries.
how to write a command for this

**Comment by BurntSushi** [comment, marked as answer]
ripgrep dowsn't search inside of zip files.

Otherwise, I think your question is better suited for some more general help forum. You'll want to give more details, including things you've tried and the format of your input.

**Comment by DanielsBerlin** [comment]
ok thanks for reply.
what if we need to search for a list of entries inside a directory of txt files

**Reply by BurntSushi** [reply]
I'm sorry, but I just do not have the time to give you the help you need. Please seek out a more general help forum friendly to beginners.

**Comment by DanielsBerlin** [comment]
i searched but i didnt find

**Reply by BurntSushi** [reply]
* https://www.linuxquestions.org/
* https://old.reddit.com/r/linuxquestions/
* https://old.reddit.com/r/learnprogramming/
* https://stackoverflow.com/

**Comment by DanielsBerlin** [comment]
can we use a command like this:
rg -aF word1, word2, word3, word4, word5
when any of the entries ( word1, word2, word3, word4, or word5) are hit in that directory , the program needs to list those matching entries
how to write that ommand?

**Reply by BurntSushi** [reply]
```
$ printf 'word1\nword2\nword3\nword4\nword5\n' > patterns
$ rg -aFf patterns ./
```
